Better namespace instructions (#99)
acend/helm-basics-training: content/en/docs/_index.md

kubectl create namespace <namespace>
```

{{% alert title="Note" color="primary" %}}
We are going to use `<namespace>` as a placeholder for your created namespace. Each time you see a `<namespace>` somewhere in a command, replace it with your chosen namespace name.
{{% /alert %}}

{{% onlyWhen rancher %}}
{{% alert title="Note" color="primary" %}}

> Namespaces created via `kubectl` have to be assigned to the correct Rancher Project in order to be visible in the Rancher web console. Please ask your teacher for this assignment. Or you can create the namespace directly within the Rancher web console.

Reviewer: ```suggestion
Namespaces created via `kubectl` have to be assigned to the correct Rancher Project in order to be visible in the Rancher web console. Please ask your trainer for this assignment or create the namespace directly within this project on the Rancher web console.
```
